### ISSUE TITLE
6.6.0 Azure Monitor data source - cannot enter API Key for Application Insights Details

### ISSUE THREAD

**Issue by kjeldlau-3shape** [grafana/grafana#21916]
**What happened**:
I am trying to configure an Azure Monitor data source in a new 6.6.0 installation. However I cannot enter the API Key for Application Insights Details in the last part of the configuration.
I had no problems in previous version 6.4.4

**What you expected to happen**:
To be able to enter the key

**How to reproduce it (as minimally and precisely as possible)**:
Start create a data source, fill in all necessary parts for Azure Monitor Details and Azure Log Analytics API Details. Press Save & Test
you will get success.
Then try to fill in API Key for Application Insights Details - it does not accept any entering in the field

**Anything else we need to know?**:
newly installed environment

**Environment**:
- Grafana version: 6.6.0
- Data source type & version: Azure Monitor
- OS Grafana is installed on: Windows
- User OS & Browser: Windows & Chrome
- Grafana plugins: N/A
- Others:

**Comment by subesokun** [comment]
The deprecated Azure Monitor plugin seems to have the same issue https://github.com/grafana/azure-monitor-datasource/issues/121

**Comment by subesokun** [comment]
After some debugging, I've noticed also that the API key gets stored in the DB `data_source` table into the `json_data` property. Not sure if it should have landed in the `secure_json_data` property like the other secrets... I've added it there now manually but still, the proxy fails to fetch the data and Application Insights reports that my API key hasn't been used yet. So it seems like the proxy is still not sending it in the headers.
